pkg/inventory/inventory_error.go

```diff
@@ -25,41 +25,31 @@ const multipleInventoryErrorStr = `Package has multiple inventory object templat
 The package should have one and only one inventory object template.
 `
 
+const namespaceInSetErrorStr = `Inventory use namespace defined in package.
+
+The inventory cannot use a namespace that is defined in the package.
+`
+
 type NoInventoryObjError struct{}
 
-func (e *NoInventoryObjError) Error() string {
+func (g NoInventoryObjError) Error() string {
```

Comment:
Generally, error types should use a pointer receiver, to allow them to be nil, even if there's no state that actually needs a pointer. Making the Error method require a pointer is a good way to force users to follow this best practice when using it.